packages/default/scss/popup/_variables.scss

```diff
@@ -35,5 +35,5 @@ $kendo-popup-text: $kendo-component-text !default;
 /// @group popup
 $kendo-popup-border: $kendo-component-border !default;
 /// Box shadow of the popup.
-/// @group popup
-$kendo-popup-shadow: 0 2px 4px 0 rgba(0, 0, 0, .03), 0 4px 5px 0 rgba(0, 0, 0, .04) !default;
+/// @group popups
+$kendo-popup-shadow: elevation(6) !default; /* 0 2px 4px 0 rgba(0, 0, 0, .03), 0 4px 5px 0 rgba(0, 0, 0, .04) */
```

Comment:
It seems that just the step is wrong here.
It should probably be step 2 - `elevation(2)`.

Reply:
Actually, It should be exactly 6 according to the design :
https://www.figma.com/file/r0H47iKFdkELP2bpX6hU9c/Effects?type=design&node-id=269-5730&mode=design&t=99JVwwSL4xE1APCN-4

Reply:
Yep, you are right.
Here comes the question why the shadow is that much different/obvious than before...

Reply:
Hey everyone. The elevation steps are defined according to the m2 guidelines where the Popup uses step 8 (X0,Y8,B10,S0 - BLACK .14; X0,Y3,B14,S0 - BLACK .12; X0,Y5,B5,S0 - BLACK .20). In our elevation system this step corresponds to depth 6, which in Default has the following values: X0,Y12,B14,S0 - black,20; X0, Y4, B16, S0 - black, .12. I agree that the shadow is much more prominent than its previous version but the steps are defined in a way that they must visible from each side (top, bottom, left and right).

Reply:
If the difference is by design/intentional then it should be fine, I guess.
It is just that we use the Popup in many other components so we need to be careful.

I'm attaching two screenshots that show how big of a difference it is:

Combobox:
![combobox](https://github.com/telerik/kendo-themes/assets/7753940/103fc207-be12-4596-8696-63f40aca3324)

Notification:
![notification](https://github.com/telerik/kendo-themes/assets/7753940/e90e5be1-1977-4001-b36f-befb5b184909)